ui/appui-react/src/appui-react/statusfields/toolassistance/ToolAssistanceField.tsx

```diff
@@ -147,6 +147,7 @@ export class ToolAssistanceField extends React.Component<
   constructor(p: ToolAssistanceFieldProps) {
     super(p);
 
+    // eslint-disable-next-line deprecation/deprecation
     const mobile = UiFramework.isMobile();
```

Comment:
Seems like `UiFramework.isMobile()` is just `ProcessDetector.isMobileBrowser`. Can't we just replace it with that here and in other places? We won't need to fix in the future and we won't need `eslint-disable` then.

Reply:
Replaced the calls.